app/assets/v2/js/grants/index.js

```diff
@@ -51,6 +51,20 @@ $(document).ready(() => {
 
   });
 
+  toggleStyle = function (style) {
+    let banner
+    if (style.bg) {
+      banner = `url("${style.bg }") center top / ${style.size || ''} ${style.color || ''} no-repeat`
+    } else {
+      banner = `url("${ style.banner_image }") center  no-repeat`
+    }
```

Comment:
we could make this cleaner with

let banner = style.bg ? 
  `url("${style.bg }") center top / ${style.size || ''} ${style.color || ''} no-repeat`:
  `url("${ style.banner_image }") center  no-repeat`